scss/_styles.scss

```diff
@@ -1,6 +1,14 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap');
 @import 'colors';
-
+@import 'mixins';
```

Comment:
This import should go in the `applications.scss`. And the Google Font and color file too

Reply:
Done 😄